chat-widget/src/components/livechatwidget/common/initWebChatComposer.ts

```diff
@@ -81,6 +83,23 @@ export const initWebChatComposer = (props: ILiveChatWidgetProps, chatSDK: any, s
         WebChatStoreLoader.store = webChatStore;
     }
 
+    const hyperlinkTextOverrideRenderer = new HyperlinkTextOverrideRenderer(hyperlinkTextOverride as boolean);
+    const markdownRenderers = [hyperlinkTextOverrideRenderer];
+    const renderMarkdown = (text: string): string => {
```

Comment:
Does this flow remains same in case of BOT, like e.g., if there is a BOT having high number of <a> tags, can it impact perf or any other area ?